File: Iniciante/1012 - Área/C#/1012 - Área.cs
using System;
using System.Globalization; // formata os valores para en-us (.)

class URI {
    static void Main() {
        double A, B, C, pi = 3.14159;
        double areaTri, areaCir, areaTrap, areaQua, areaRet;

        string[] calculo = Console.ReadLine().Split(' ');
        // separa os valores para cada posição do array
        A = double.Parse(calculo[0]);
        B = double.Parse(calculo[1]); // cada variável recebe seu respectivo valor
        C = double.Parse(calculo[2]);

        areaTri = (A*C)/2;
        areaCir = pi*Math.Pow(C, 2);
        areaTrap = ((A+B)*C)/2;
        areaQua = Math.Pow(B, 2);
        areaRet = A*B;

        Console.WriteLine("TRIANGULO: " + areaTri.ToString("0.000", System.Globalization.CultureInfo.InvariantCulture));
        Console.WriteLine("CIRCULO: " + areaCir.ToString("0.000", System.Globalization.CultureInfo.InvariantCulture));
        Console.WriteLine("TRAPEZIO: " + areaTrap.ToString("0.000", System.Globalization.CultureInfo.InvariantCulture));
        Console.WriteLine("QUADRADO: " + areaQua.ToString("0.000", System.Globalization.CultureInfo.InvariantCulture));
        Console.WriteLine("RETANGULO: " + areaRet.ToString("0.000", System.Globalization.CultureInfo.InvariantCulture));
    }
}

File: Iniciante/1012 - Área/Java/1012 - Área.java
import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws Exception {
    	// cria um objeto do tipo Scanner
    	Scanner leitor = new Scanner(System.in).useLocale(Locale.US);
    	
    	// recebe os valores na mesma linha
    	float a = leitor.nextFloat();
    	float b = leitor.nextFloat();
    	float c = leitor.nextFloat();
    	
    	// calcula e apresenta o resultado de cada caso
    	System.out.printf("TRIANGULO: %.3f\n", (a*c)/2);
    	System.out.printf("CIRCULO: %.3f\n", 3.14159*Math.pow(c,2));
    	System.out.printf("TRAPEZIO: %.3f\n", ((a+b)*c)/2);
    	System.out.printf("QUADRADO: %.3f\n", Math.pow(b, 2));
    	System.out.printf("RETANGULO: %.3f\n", a*b);
    	
    }
}